_includes/usa_identifier.html

```diff
@@ -0,0 +1,67 @@
+{% assign identifier = site.data.usa_identifier %}
+<div class="usa-identifier">
+  <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
+    <div class="usa-identifier__container">
+      <div class="usa-identifier__logos">
+        <a href="https://www.gsa.gov/" class="usa-identifier__logo">
+          <img class="usa-identifier__logo-img" src="{{ site.baseurl }}{{ identifier.agency_logo }}" alt="GSA logo" role="img" />
+        </a>
+      </div>
+      <div class="usa-identifier__identity text-base-lightest" aria-label="Agency description">
+        <p class="usa-identifier__identity-domain">18F Visual Identity</p>
```

Comment:
[nitpick] this could be {{ identifier.site_name}} but that idea just occurred to me looking over your PR and other guides don't do it so nonblocking/up to you!

Reply:
Great catch, thank you!